internal/components/receivers_test.go

```diff
@@ -79,21 +162,82 @@ func TestDefaultReceivers(t *testing.T) {
 				return cfg
 			},
 		},
+		{
+			receiver:     "prometheus_exec",
+			skipLifecyle: true, // Requires running a subproccess that can not be easily set across platforms
+		},
+		{
+			receiver: "receiver_creator",
+		},
+		{
+			receiver: "redis",
+		},
 		{
 			receiver: "sapm",
 		},
 		{
 			receiver: "signalfx",
 		},
+		{
+			receiver: "prometheus_simple",
+		},
 		{
 			receiver: "splunk_hec",
 		},
+		{
+			receiver: "statsd",
+		},
+		{
+			receiver:     "wavefront",
+			skipLifecyle: true, // Depends on carbon receiver to be running correctly
+		},
+		{
+			receiver:     "windowsperfcounters",
+			skipLifecyle: runtime.GOOS != "windows", // Only works on windows based machines
+		},
 		{
 			receiver: "zipkin",
 		},
+		{
+			receiver:     "zookeeper",
```

Comment:
This requires a follow up issue to make the shutdown work without issue with the default configuration.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6979